doc/installation/linux_install.rst

```diff
@@ -129,3 +117,40 @@ If you want to run the ``nest`` executable or use the ``nest`` Python module wit
 The script ``/install/path/bin/nest_vars.sh`` can be sourced in ``.bashrc`` and will set these paths for you. This also allows to switch between NEST installations in a convenient manner.
 
 
+Environment variables
+---------------------
+
+There are several environment variables that describe where components of the NEST installation can be found. In particular when installing to a custom directory, it is typically necessary to explicitly set these variables, so that your operating system can find the NEST binaries and libraries.
```

Comment:
```suggestion
A number of environment variables are used to specify where the components of a NEST installation are found. In particular when installing to a custom directory, it is typically necessary to explicitly set these variables, so that your operating system can find the NEST binaries, its libraries and custom extension modules.
```